### PR TITLE
Change list screen

### PR DESCRIPTION
**Pull request by EntyR**

### CODE REVIEW

**Comment by AlexeyKolosov** [comment]
1. Please add top margin, because now cards are under status bar
2. Please add `SnapHelper` for this recycler view to allow central card be right in the center after scroll
3. Please make cards squared
4. Please make sure that first and last cards can be in the center
5. Please make sure that all screens (of any size) display one card in the center and two half-cards (no more than 3 cards)

File: app/src/main/java/com/harman/roomdbapp/app/ui/fragments/RandomNumbersList.kt [hunk 1]
```diff
@@ -27,18 +33,42 @@ class RandomNumbersList : Fragment() {
...
+            val widgetAdapter = WidgetAdapter(requireContext(), binding.rvWidgetList.width)
+            binding.rvWidgetList.apply {
+                snapHelper.attachToRecyclerView(this)
+                layoutManager = lManager
+                adapter = widgetAdapter
+            }
+            widgetAdapter.submitList(FakeRepository.repository)
+        }
+
+        binding.rvWidgetList.doOnPreDraw {
```
Comment (AlexeyKolosov) [inline app/src/main/java/com/harman/roomdbapp/app/ui/fragments/RandomNumbersList.kt]: please remove if it does nothing

File: app/src/main/java/com/harman/roomdbapp/app/ui/fragments/RandomNumbersList.kt [hunk 2]
```diff
@@ -27,18 +33,42 @@ class RandomNumbersList : Fragment() {
...
+        binding.rvWidgetList.post {
+            val snapHelper = LinearSnapHelper()
+            val lManager = CenterZoomLayoutManager(requireContext(), LinearLayoutManager.HORIZONTAL)
+            val widgetAdapter = WidgetAdapter(requireContext(), binding.rvWidgetList.width)
+            binding.rvWidgetList.apply {
+                snapHelper.attachToRecyclerView(this)
+                layoutManager = lManager
+                adapter = widgetAdapter
+            }
+            widgetAdapter.submitList(FakeRepository.repository)
```
Comment (AlexeyKolosov) [inline app/src/main/java/com/harman/roomdbapp/app/ui/fragments/RandomNumbersList.kt]: as you have already tested this, please remove "FakeRepository" and keep here only one item for a while

File: app/src/main/java/com/harman/roomdbapp/app/ui/fragments/RandomNumbersList.kt (hunk 1 above)
Comment (EntyR) [reply app/src/main/java/com/harman/roomdbapp/app/ui/fragments/RandomNumbersList.kt]: done

File: app/src/main/java/com/harman/roomdbapp/app/ui/fragments/RandomNumbersList.kt (hunk 2 above)
Comment (EntyR) [reply app/src/main/java/com/harman/roomdbapp/app/ui/fragments/RandomNumbersList.kt]: done

**Review by AlexeyKolosov — APPROVED** [review]

**Pull request merged**